src/Component/Action/PlaceHolderAction.php

```diff
@@ -13,13 +13,10 @@
 
 namespace Sylius\Component\Resource\Action;
 
-final class PlaceHolderAction
-{
-    /**
-     * @param mixed $data
-     */
-    public function __invoke($data = null): mixed
+class_exists(\Sylius\Resource\Action\PlaceHolderAction::class);
+
+if (false) {
+    final class PlaceHolderAction extends \Sylius\Resource\Action\PlaceHolderAction
```

Comment:
Is it alright to extend `final` class?

Reply:
Yes, it's strange but it's done like this. I already did that based on API Platform bc layer and it works.